File: bee/simples/2863.go
package main

import (
	"fmt"
)

func menore() error {
	menor := 9999999999999999999999999999.9
	var voltas int
	_, err := fmt.Scanf("%d", &voltas)
	if err != nil {
		return err
	}

	for i := 0; i < voltas; i++ {
		var num float64
		fmt.Scanf("%f", &num)
		if num < menor {
			menor = num
		}
	}

	fmt.Printf("%.2f\n", menor)

	return nil
}

func main() {
	for {
		if err := menore(); err != nil {
			break
		}
	}
}

File: bee/simples/2863.py
def menor():
    menor = 999999999999999999999999999999999999999999999

    voltas = int(input())

    for i in range(voltas):
        num = float(input())
        if num < menor:
            menor = num

    print(f"{menor:.2f}")

while True:
    try:
        menor()
    except EOFError:
        break
